connections.md

```diff
@@ -2,254 +2,7 @@
 
 A Connection is the practical application of a relationship in DID Communication. Having a connection means that each party in the relationship has a DID for the other parties, and parties can communicate securely using the keys and endpoints within each DID Document.
 
-In order to establish a new connection, we use the OutOfBand Protocol and the DID Exchange Protocol.
+In order to establish a new connection, Simply exchange a new message between parties. Knowing the DID of the other parties does not indicate any level of trust. Using the connection to establish a foundation of trust is the next step.
 
 
 
-#### OutOfBand Protocol
-
-Invitations are usually the first message passed between parties. They are passed out of band, such as in a QR code or a link that can either open in an app or in a browser containing getting started instructions.
-
-The OutOfBand Protocol exists as an independent protocol to allow for the invitation recipient to respond with any of the supported protocols.
-
-A Public DID functions as a standing invitation. The information within the resolved DID Document serves the same function as the information passed in OutOfBand protocol messages.
-
-TODO: Include details of the protocol.
-
-##### Ephemeral Mode
-
-Some interactions do not need an established connection to complete their purpose. These interactions occur in _ephemeral mode_ and use OutOfBand protocol messages for the bulk of the interaction.
-
-TODO: Include more details about ephemeral mode.
-
-#### DID Exchange Protocol
-
-##### 1. Exchange Request
-
-The exchange request message is used to communicate the DID document of the _invitee_ to the _inviter_ using the provisional service information present in the _invitation_ message.
-
-If needed, the _invitee_ will provision a new DID according to the DID method spec. The _invitee_ may also use a previously provisioned DID.
-
-The DID is presented in the exchange_request message as follows:
-
-TODO: Add clear details about initial_state usage?
-
-Example
-
-```json
-{
-  "@id": "5678876542345",
-  "@type": "https://didcomm.org/didexchange/2.0/request",
-  "~thread": { "pthid": "<id of invitation>" },
-  "label": "Bob",
-  "did": "<did invitee is sending to inviter>"
-}
-```
-
-###### Attributes
-
-* The `@type` attribute is a required string value that denotes that the received message is an exchange request.
-* The [`~thread`](../../concepts/0008-message-id-and-threading/README.md#thread-object) decorator MUST be included:
-  * It MUST include the ID of the parent thread (`pthid`) such that the `request` can be correlated to the corresponding `invitation`. More on correlation [below](#correlating-requests-to-invitations).
```

Comment:
parent thread will need to be defined elsewhere in the spec.

Reply:
https://github.com/decentralized-identity/didcomm-messaging/issues/95#issuecomment-692226190